pegasus/helper_modules/dashboard.rb

```diff
@@ -0,0 +1,93 @@
+#
+# A set of utilities that provide access to the current user, their
+# authentication state, and other DASHBOARD_DB access.
+#
+module Dashboard
+
+  # Get the dashboard database
+  # The static constant is wrapped here so it's easy to stub for tests.
+  # @returns [Database]
+  def self.db
+    DASHBOARD_DB
+  end
+
+
+  def self.admin?(user_id)
+    !!db[:users].where(id: user_id, admin: true).first
```

Comment:
One option is square-bracket notation, which (while a bit confusing) does immediately execute on the database without needing to be resolved by `first` or `all`:

```ruby
!!db[:users][id: user_id, admin: true]
```